### PR TITLE
Release 2.4.0

### PR DESCRIPTION
**Pull request by m1r0**
### Changes proposed in this Pull Request

* Bump the version
* Update the changelog
* Update the pot file
* Update the packages
* Some small maintenance changes

### Testing instructions

* Go to Plugins and make sure the version is `2.4.0`.
* Make sure the plugin works as before.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+16
```
Comment (donnapep) [inline .nvmrc]: Just curious as to why this is 16 and not 18 like Sensei?

File: readme.txt [hunk 2]
```diff
@@ -1,10 +1,10 @@
 === Sensei LMS Certificates ===
 Contributors: automattic, alexsanford1, donnapep, jakeom, gikaragia, renathoc, yscik, m1r0, aaronfc
 Tags: certificates, course certificate, sensei lms
-Requires at least: 5.6
-Tested up to: 6.0
+Requires at least: 6.0
+Tested up to: 6.2
 Requires PHP: 7.0
```
Comment (donnapep) [inline readme.txt]: We should bump this too to match Sensei.

Comment (m1r0) [reply readme.txt]: Good idea. Updated in https://github.com/woocommerce/sensei-certificates/pull/337/commits/ae624b3f52522a147951791c19883d1869281c2e.

File: .nvmrc (hunk 1 above)
Comment (m1r0) [reply .nvmrc]: Yeah, that sounds convenient. I've switched it to 18 but I had to update the packages. 3c43ac8436c0b36e1ca56d72e1db8a3520c6d8d0 fb45e4a2596c20eb64b4e245d018e2b12352da52

**Review by donnapep — APPROVED** [review]

**Pull request merged**